source/layout.html

```diff
@@ -78,7 +78,9 @@ <h1 class="custom-page-header page-title has-dash">{{ page.name }}</h1>
             <div class="custom-page-content">{{ page_content | paragraphs }}</div>
           </div>
         {% else %}
-          {{ page_content }}
+          <div>

```

Comment:
would love to give this some sort of class name that describes it's purpose - i have a feeling we're going to inevitably want to target it.

Reply:
Are you sure about this?  I originally had it as `main-content` for both the custom page and non, but it seemed to not make sense and instead settled on `main` which is the parent to all of this on line 65